ansible-container/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rhel7
+
+MAINTAINER Aaron Weitekamp <aweiteka@redhat.com>
+
+RUN yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+
+# Not sure if all of these packages are necessary
+# only git and ansible are known requirements
+RUN yum install -y --enablerepo rhel-7-server-extras-rpms net-tools bind-utils tmux git ansible
+
+RUN git clone https://github.com/openshift/openshift-ansible.git /opt/openshift-ansible
```

Comment:
It would be better to use the `ADD` directive here.

1. No need to clone when it's local.
1. We want to control the packaging, not just pull from master.

```
ADD / /opt/openshift-ansible/
```